cpu/cc2538/periph/adc.c

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+#include "board.h"
+#include "cpu.h"
+#include "sched.h"
+#include "thread.h"
+#include "periph_conf.h"
+#include "cpu_conf.h"
+#include "periph/adc.h"
+#include "cc2538.h"
+
+enum{
+
+    EOC = 1,  
+    STSEL = 3, 
+};
+
+int adc_init(adc_t line)
+{
+    /* make sure the given ADC line is valid */
+    if (line >= ADC_NUMOF) {
+        return -1;
+    }
+
+    cc2538_soc_adc_t *adcaccess;
```

Comment:
Perhaps it would be useful also to configure the pin as analogue `IOC_OVERRIDE_ANA`